pwiz_tools/Skyline/Model/Import.cs

```diff
@@ -1976,6 +1976,11 @@ private static bool TrySplitColumns(string line, char sep, out string[] columns)
         private static Type GetColumnType(string value, IFormatProvider provider)
         {
             double result;
+            var quote = @"""";

```

Comment:
Looks like my fault for sure in trying to handle international inputs regardless of locale. I think the correct fix is actually to get rid of TrySplitColumns and use ParseDsvFields instead. It is what gets used in the end, and it already has logic for dealing with quoted fields.

Reply:
Can we go with this patch for now and enter an issue to do the more thorough fix later?

Reply:
It is not a lot of work to try what I suggested. It is like changes to 3 lines. If it passes all existing tests and your new test, it is a way better change that would fix other potential bugs.

Reply:
Fair enough.

Reply:
Actually I think it's sufficient to change just one line within TrySplitColumns rather than replacing it- just change line.Split with line.ParseDsvColumns.  This accommodates the data set that alerted us to the issue, and the previously modified test.